<launch>

    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />    <!-- 开启机器人底层相关节点  -->
   
    <include file="$(find turn_on_wheeltec_robot)/launch/turn_on_wheeltec_robot.launch" />

    <include file="$(find rplidar_ros)/launch/rplidar.launch" />    <!-- 激光雷达开启 -->

    <node name="xjh_node" pkg="xjh_learn" type="xjh_node" output="screen">
        <param name="move_speed"  value="0.05"/>
        <param name="spin_speed"  value="0.24"/>
        <param name="scale_factor_x" value="1.000000"/>
        <param name="scale_factor_z" value="1.000000"/>
    </node>

    <!-- 设置需要用于导航的地图  -->
    <arg name="map_file" default="$(find turn_on_wheeltec_robot)/map/WHEELTEC.yaml"/>
    <node name="map_server_for_test" pkg="map_server" type="map_server" args="$(arg map_file)">
    </node>

    <!-- 开启用于导航的自适应蒙特卡洛定位amcl_omni  运动方式是omni old amcl_omni-->
    <include file="$(find turn_on_wheeltec_robot)/launch/include/amcl.launch" />

   
    <!-- 开启dwa_local_planner导航算法-一般用于差速车-->
    <include file="$(find turn_on_wheeltec_robot)/launch/include/dwa_local_planner.launch" />
</launch>